src/main/java/io/visual_regression_tracker/sdk_java/VisualRegressionTrackerConfig.java

```diff
@@ -11,4 +11,5 @@
     private String apiKey;
     private String branchName;
     private Boolean enableSoftAssert;
+    private String ciBuildId;
```

Comment:
as this is optional field
there should be an option to create config with and without such
please, add default value as `null`

Reply:
In last changes I've added requiredArgsContructor and default values for configuration fields.